starboard/linux/shared/pre_app_recommendation_service.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef STARBOARD_LINUX_SHARED_PRE_APP_RECOMMENDATION_SERVICE_H_
+#define STARBOARD_LINUX_SHARED_PRE_APP_RECOMMENDATION_SERVICE_H_
+
+// Omit namespace linux due to symbol name conflict.
```

Comment:
There should not be any symbol name conflicts here ?

Reply:
Conflict with macro 'linux'